src/checkout_fe.tf

```diff
@@ -83,3 +83,26 @@ module "checkout_cdn" {
 
   tags = var.tags
 }
+
+resource "azurerm_application_insights_web_test" "checkout_fe_web_test" {
```

Comment:
since we added health check path in function u can use `Health Check Status` metrics, in this way u'll ping all function instances

Reply:
This test is not for the function app. The goal of this test is to monitor if the front-end exposes resources, that is, if the GET dns/index.html responds 200.

Reply:
thanks, my mistake I saw function